subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/samples/UserGuideSamplesRunner.groovy

```diff
@@ -134,8 +135,8 @@ class UserGuideSamplesRunner extends Runner {
             File rootProjectDir = temporaryFolder.testDirectory.file(singleRun.subDir)
             if (rootProjectDir.exists()) {
                 def delete = new Delete()
-                delete.dir = rootProjectDir
-                delete.includes = "**/.gradle/** **/build/**"
+                delete.includeEmptyDirs = true
+                delete.addFileset(new FileSet(dir: rootProjectDir, includes: "**/.gradle/** **/build/**"))
```

Comment:
We are directly invoking the Ant task here: `AntUtil.execute(delete)`. It's not a practice we promote in Gradle as it would mean calling the `execute()` method explicitly. Other than that we don't we have a method in `GFileUtils` or FileUtils` that accepts a pattern.